### PR TITLE
EG-1225: Add a link to cordapp-overview, fix some formatting

### PR DESCRIPTION
**Pull request by edward-prosser**
- Fixed formatting around the list.
- Tidied some wording.
- Corrected some terminology (Cordapp -> CorDapp etc.)
- Fixes https://r3-cev.atlassian.net/browse/EG-1225
- Confirmed with Valerio (issue reporter) that fix is valid.

Signed-off-by: Ed Prosser <edward.prosser@r3.com>

### CODE REVIEW

File: content/en/docs/corda-os/4.4/release-notes.md [hunk 1]
```diff
@@ -19,31 +19,42 @@ title: Release notes
...
 
 ## Corda 4.4
 
-Corda 4.4 lays the foundation of a new open-core approach for the Corda codebase. This involved a refactoring of the main functional components of Corda. Please consult cordapp-overview.rst to get an overview of the practical impact on CorDapp development.
+Corda 4.4 lays the foundation of a new open-core approach for the Corda codebase. This involved a refactoring of the
+main functional components of Corda. Please consult [the CorDapp overview](cordapp-overview.md/) to get an overview of
+the practical impact on CorDapp development.
 
-Furthermore, Corda 4.4 introduces improvements to the flow framework API, a new diagnostic `ServiceHub` call and includes a number of security enhancements.
+Furthermore, Corda 4.4 introduces improvements to the flow framework API, a new diagnostic `ServiceHub` call and includes
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.4/release-notes.md:34]: Shouldn't that be
Furthermore, Corda 4.4 introduces improvements to the flow framework API and a new diagnostic `ServiceHub` call, and includes...

File: content/en/docs/corda-os/4.4/release-notes.md [hunk 2]
```diff
@@ -19,31 +19,42 @@ title: Release notes
...
+Welcome to the Corda 4.4 release notes. Please read these carefully to understand what’s new in this release and how the
+features can help you. Just as prior releases have brought with them commitments to wire and API stability, Corda 4.4
+comes with those same guarantees. States and apps valid in Corda 3.0 are usable in Corda 4.4.
 
 
 
 ## Corda 4.4
 
-Corda 4.4 lays the foundation of a new open-core approach for the Corda codebase. This involved a refactoring of the main functional components of Corda. Please consult cordapp-overview.rst to get an overview of the practical impact on CorDapp development.
+Corda 4.4 lays the foundation of a new open-core approach for the Corda codebase. This involved a refactoring of the
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.4/release-notes.md:30]: In line with style guide discussions for open source (as opposed to open-source), I feel we should make open-core open core too?

**Comment by ivanterziev-r3** [comment]
@edward-prosser This page is duplicated in .../corda-docs/content/en/docs/corda-enterprise/4.4/release-notes.md - there is the same issue there so essentially the same changes need to go there as well.

File: content/en/docs/corda-os/4.4/release-notes.md [hunk 3]
```diff
@@ -19,31 +19,42 @@ title: Release notes
...
 
 ### Changes for developers in Corda 4.4
 
 
 #### Flows API improvements
 
-Corda 4.4 introduces a new `FlowLogic.await` API that allows a CorDapp developer to suspend their flow when executing user-defined long-running operations (e.g. call-outs to external services). This prevents these long-running operations from blocking the flow thread, allowing other flows to progress in the interim. Previously, these operations had to be executed synchronously, blocking the flow thread.
+Corda 4.4 introduces a `FlowLogic.await` API that allows a CorDapp developer to suspend their flow when executing
+user-defined long-running operations such as call-outs to external services. The `FlowLogic.await` API prevents these
+long-running operations from blocking the flow thread, allowing other flows to progress. Previously, these operations
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.4/release-notes.md:45]: a few missing commas here in my view (a legacy issue surely - "engineers write docs")

File: content/en/docs/corda-os/4.4/release-notes.md [hunk 4]
```diff
@@ -19,31 +19,42 @@ title: Release notes
...
+had to be executed synchronously, blocking the flow thread.
 
-The CorDapp developer can decide whether to run these asynchronous flow operations in a dedicated thread pool, or to handle the threading themselves directly.
+The CorDapp developer can decide whether to run these asynchronous flow operations in a dedicated thread pool, or to
+handle the threading themselves directly.
 
-Note that as before, the flow framework suspends automatically for certain operations (e.g. when waiting to receive a message from a counterparty). These suspensions do not have to be triggered explicitly.
+Note that as before, the flow framework suspends automatically for certain operations (e.g. when waiting to receive a
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.4/release-notes.md:51]: e.g. > for example, ...

File: content/en/docs/corda-os/4.4/release-notes.md (hunk 2 above)
Comment (edward-prosser) [inline content/en/docs/corda-os/4.4/release-notes.md:30]:
```suggestion
Corda 4.4 lays the foundation of a new open core approach for the Corda codebase. This involved a refactoring of the
```

File: content/en/docs/corda-os/4.4/release-notes.md (hunk 1 above)
Comment (edward-prosser) [reply content/en/docs/corda-os/4.4/release-notes.md:34]:
```suggestion
Furthermore, Corda 4.4 introduces improvements to the flow framework API and a new diagnostic `ServiceHub` call, and includes
```

File: content/en/docs/corda-os/4.4/release-notes.md (hunk 4 above)
Comment (edward-prosser) [reply content/en/docs/corda-os/4.4/release-notes.md:51]:
```suggestion
Note that as before, the flow framework suspends automatically for certain operations (for example, when waiting to receive a
```

File: content/en/docs/corda-os/4.4/release-notes.md (hunk 3 above)
Comment (edward-prosser) [inline content/en/docs/corda-os/4.4/release-notes.md:45]:
```suggestion
Corda 4.4 introduces a `FlowLogic.await` API, allowing a CorDapp developer to suspend a flow when executing
user-defined long-running operations, such as call-outs to external services. The `FlowLogic.await` API prevents these
long-running operations from blocking the flow thread, allowing other flows to progress. Previously, these operations
```

**Pull request merged**